### PR TITLE
[ADD] Helper migration scripts

### LINKED ISSUE #224: Write a script to launch the migration script on all databases
Write a script to launch the migration scrip on all databases
- script is at the root of vertical-cooperative

### PR DESCRIPTION
**Pull request by carmenbianca**
## Description

Put these files in the repo. I slightly patched them to work from any CWD.

Fixes #224 

## Odoo task (if applicable)

n/a

### CODE REVIEW

File: rename_all_deprecated_modules.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+def now():
+    return datetime.datetime.now().strftime("%Y-%m-%dT%H-%m")
+
+
+def main():
+    os.chdir(os.path.dirname(__file__))
+    logging.basicConfig(level=logging.INFO)
+    databases = filter_databases(all_databases())
+    for database in databases:
+        new_database = f"{database}-renamedep-{now()}"
```
Comment (robinkeunen) [inline rename_all_deprecated_modules.py]: Do we really need a backup ? Or is the idea to have `xxx-komunigi-test` always on old modules ?

Comment (carmenbianca) [reply rename_all_deprecated_modules.py]: The convenience of creating a new database is that this script becomes repeatable. If it failed, you don't have to delete + recreate `xxx-komunigi-test`. You can simply do your changes and run the script again.

Comment (robinkeunen) [reply rename_all_deprecated_modules.py]: 🆗

**Review by robinkeunen — APPROVED** [review]

**Comment by robinkeunen** [comment]
Leaving @huguesdk the opportunity to take a look.

**Comment by carmenbianca** [comment]
Going to remove the datetime logic as agreed in mini-call.

**Comment by victor-champonnois** [comment]
@carmenbianca should this be merged ?

**Comment by carmenbianca** [comment]
@victor-champonnois good to merge.

**Pull request merged**